src/parameterizations/lateral/MOM_internal_tides.F90

```diff
@@ -779,9 +781,36 @@ subroutine refract(En, cn, freq, dt, G, US, NAngle, use_PPMang)
   real :: cn_subRO        ! A tiny wave speed to prevent division by zero [L T-1 ~> m s-1]
   integer :: is, ie, js, je, asd, aed, na
   integer :: i, j, a
+  real :: wgt1, wgt2
+  real :: eps
 
   is = G%isc ; ie = G%iec ; js = G%jsc ; je = G%jec ; na = size(En,3)
   asd = 1-stencil ; aed = NAngle+stencil
+  eps=1.0e-20 * US%m_s_to_L_T
+
+  do i=is-1,ie ; do j=js-1,je
```

Comment:
This loop extent would make sense for variables that are discretized at a corner point, but not at a u-point. I suspect that the correct extent here should be
`do I=is-1,ie ; do j=js,je`
 based on how cn_u is used in line 845 below.

Reply:
fixed